test/propagatortest.jl

```diff
@@ -0,0 +1,33 @@
+using Base.Test
```

Comment:
I get an error if I exclude that line on the lines : 

```ERROR: @test_approx_eq_eps not defined ```

so I have included it.